packages/inter-protocol/scripts/start-local-chain.sh

```diff
@@ -23,7 +23,7 @@ if [ -z "$WALLET" ]; then
     exit 1
 fi
 
-WALLET_BECH32=$(agd keys show "$WALLET" --output json | jq -r .address)
+WALLET_BECH32=$(agd keys --keyring-backend=test show "$WALLET" --output json | jq -r .address)
```

Comment:
@samsiegart this should make life easier for you.

Reply:
thanks @mhofman for making this script more WSL friendly